src/workerd/server/tests/container-client/test.js

```diff
@@ -197,6 +220,15 @@ export const testBasics = {
   },
 };
 
+// Test exit code monitor functionality
+export const testExitCode = {
+  async test(_ctrl, env) {
+    const id = env.CONTAINER.idFromName('testExitCode');
+    const stub = env.CONTAINER.get(id);
```

Comment:
This is actually wrong. The test should be as following.

env.MY_CONTAINER rather than env.CONTAINER

Reply:
We really need to actually run this test in CI. Can we set it up with a mock backend?